common/src/main/java/com/wynntils/handlers/container/ContainerQueryHandler.java

```diff
@@ -46,7 +45,7 @@ public void runQuery(ContainerQueryStep firstStep) {
         if (currentStep != null) {
             // Only add if it is not already enqueued
             if (queuedQueries.stream()
-                    .filter(query -> query.getName().equals(firstStepName))
+                    .filter(query -> query.getName().equals(firstStep.getName()))
```

Comment:
What a typo! I am sure this is obvious after a second look...

Reply:
Yeah, really! Maybe we should rename it, like storedNameOfFirstStep....